style.css

```
@@ -1,279 +1,396 @@
 @import url(./global.css);
+@import url(./responsive-variables.css);
 
-:root {
-    /* font-size: 16px; */
-    font-size: 0.83333333vw;
-    /*
-    => 가변 사이즈로
-       브라우저 너비 1920=> 100vw  
-      16/1920*100 =0.83333...
-    */
-    /* 컨테이너 관련 변수 */
-    --container-width: 70rem;
-    --container-margin: 0 auto;
-}
-
+/* 공통(모바일) */
 body {
     background-color: var(--background-color);
     width: 100vw;
     overflow-x: hidden;
 }
 
+.container {
+    margin: var(--container-margin);
+    max-width: var(--container-width);
+    width: 100%;
+}
+
 .page-wrapper {
     background-color: var(--article-background-color);
     color: var(--text-color);
     max-width: 1920px;
+    width: 100%;
     margin: 0 auto;
 }
 
-.container {
-    /*w1120 -> 추천해주신 container로 이름 변경*/
-    margin: var(--container-margin);
-    width: var(--container-width);
-}
-
-/* Header */
+/* header */
 header {
-    height: 4.375rem;
-    padding: 0.5625rem 0;
-    /*기본 체크사항 4 설명해주신 대로 동작하도록 수정*/
     display: flex;
     justify-content: center;
     align-items: center;
-    /*sprint2 추가 내용*/
     position: sticky;
     top: 0;
     background-color: var(--white-color);
+    height: 70px;
+    padding: 9px 0;
 }
 
 nav {
     display: flex;
-    justify-content: space-between;
     align-items: center;
-    flex: 0 1 70rem;
-    height: 3.1875rem;
-    margin: 0 200px;
-}
-
-.nav__logo img {
-    display: inline-block;
-    width: 9.5625rem;
+    height: 51px;
+    margin: var(--nav-margin);
+    justify-content: space-between;
+    flex-grow: 1;
 }
 
 .nav__button {
     display: inline-block;
     background-color: var(--primary-color);
-    color: var(--white-color);
+    color: var(--secondary-gray100);
     border-radius: 8px;
     font-weight: 600;
     text-align: center;
-    padding: 0.75rem 1.4375rem;
-    height: 3rem;
-    width: 8rem;
-    font-size: 1rem;
-    line-height: 1.625rem;
+    padding: 12px 23px;
+    line-height: 26px;
+    height: 48px;
+    width: 128px;
 }
 
-.nav__button:hover {
+.nav__button:hover,
+.hero__text button:hover {
     background-color: var(--primary-hover);
 }
 
-/* Main */
-section.main__hero {
+/* main 공통 */
+section>article img {
```

여기서도 띄어쓰기 한칸씩 해줍시다! 
포맷팅할때 기준이 햇갈리신다면, prettier 기본 규칙 참고해서 좋은 포맷팅을 유지할수있도록 해볼까요?

[참고](https://velog.io/@2_hyeonju/Prettier-ESLint-%EC%99%84%EC%A0%84-%EC%A0%95%EB%B3%B5)

```suggestion
section > article img {
```